Lux/lux_react/src/components/panes/HomePane.jsx

```diff
@@ -52,19 +58,72 @@ function HomePane({ dimensions, panelSize, panelJSON, activeGroups, onWidgetGrou
         return breakpoints;
     }, [containerWidth, panelSize]); // Only recalculate when width changes
 
-    // Filter out any group that has image picker widgets for the Home pane
-    const nonImageGroups = activeGroups.filter(group => {
-        // Skip source/target image groups entirely
-        if (group.name.toLowerCase().includes('source') ||
-            group.name.toLowerCase().includes('target') ||
-            group.name === 'SOURCE_IMAGE_GROUP' ||
-            group.name === 'TARGET_IMAGE_GROUP'
-            ) 
-            {
-            return false;
+    // Find groups that have image picker widgets
+    const imageGroups = activeGroups.filter(group => {
+        if (group.widgets) {
+            return group.widgets.some(widgetName => {
+                try {
+                    const widgetJSON = window.module?.get_widget_JSON(widgetName);
+                    const widget = JSON.parse(widgetJSON);
+                    return widget?.tool === 'image';
+                } catch (error) {
+                    return false;
+                }
+            });
         }
+        return false;
+    });
 
-        // For other groups, check if they have image pickers
+    // Get the active/selected image from any image group
```

Comment:
It's not necessary here to look for image pickers within the scene file. Just let the image picker widget include the name of the image and the thumbnail, and let it live within the ordinary flow of widgets.